web-registry/src/pages/Dashboard/MyBridge/MyBridge.BridgableTable.tsx

```diff
@@ -0,0 +1,151 @@
+import { useState } from 'react';
+import { Box } from '@mui/material';
+import { quantityFormatNumberOptions } from 'config/decimals';
+import { ELLIPSIS_COLUMN_WIDTH, tableStyles } from 'styles/table';
+
+import { BlockContent } from 'web-components/lib/components/block-content';
+import OutlinedButton from 'web-components/lib/components/buttons/OutlinedButton';
+import BridgeIcon from 'web-components/lib/components/icons/BridgeIcon';
+import EmptyCartIcon from 'web-components/lib/components/icons/EmptyCartIcon';
+import {
+  ActionsTable,
+  DEFAULT_ROWS_PER_PAGE,
+  TablePaginationParams,
+} from 'web-components/lib/components/table/ActionsTable';
+import InfoTooltipWithIcon from 'web-components/lib/components/tooltip/InfoTooltipWithIcon';
+import { formatDate, formatNumber } from 'web-components/lib/utils/format';
+
+import { useLedger } from 'ledger';
+
+import { AccountLink, Link } from 'components/atoms';
+import WithLoader from 'components/atoms/WithLoader';
+import { NoCredits } from 'components/molecules';
+import { useEcocredits } from 'hooks';
+
+import {
+  AMOUNT_BRIDGABLE_TOOLTIP,
+  BRIDGE_ACTION,
+  CREDIT_BATCH_TOOLTIP,
+  NO_BRIDGABLE_CREDITS,
+} from './MyBridge.constants';
+import { WithLoaderStyles } from './MyBridge.styles';
+import { BreakText, BreakTextEnd, GreyText } from './MyBridge.utils';
+
+export const BridgableTable = (): JSX.Element => {
+  const { wallet } = useLedger();
+
+  const [paginationParams, setPaginationParams] =
+    useState<TablePaginationParams>({
+      page: 0,
+      rowsPerPage: DEFAULT_ROWS_PER_PAGE,
+      offset: 0,
+    });
+
+  const { credits, isLoadingCredits } = useEcocredits({
+    address: wallet?.address,
+    paginationParams,
+  });
+
```

Comment:
We will filter this by `C03` class eventually, right? Maybe add a TODO for this?

Reply:
Thanks!